samples/highcharts/demo/polar-advanced/demo.js

```diff
@@ -0,0 +1,479 @@
+const
+    monthExtremes = { min: 0, max: 26 },
+    weekExtremes = { min: 1, max: 5 },
+    paneOpeningAngles = { startAngle: 40.5, endAngle: 319.5 },
+    noLabelProp = { labels: { enabled: false } },
+    specialSeriesProps = {
+        showInLegend: false,
+        groupPadding: 0,
+        pointPadding: 0
+    },
+    toggleableGradient = {
+        pattern: undefined,
+        radialGradient: [1, 0.25, 0.1],
+        stops: [
+            [0, '#1f1836'],
+            [1, '#45445d']
+        ]
+    },
+    mouseOut = function () {
+        const chart = this.series.chart;
+        chart.setMidPaneBg({
+            backgroundColor: toggleableGradient,
+            outerRadius: '75%'
+        });
+        chart.subtitle.element.style.opacity = 1;
+    },
+    colors = Highcharts.getOptions().colors,
+    asHcColor = hexval => new Highcharts.Color(hexval),
+    // Declaring colors used more than once
+    hcColor0 = asHcColor(colors[0]),
+    hcColor1 = asHcColor(colors[1]),
+    hcColor5 = asHcColor(colors[5]),
+    hcColor8 = asHcColor(colors[8]),
+    hcColor9 = asHcColor(colors[9]),
+    data = JSON.parse(document.getElementById('data').innerHTML),
+    scoreData = data[3],
+    countries = ['Ulambaator', 'Sofia', 'Asmara'],
+    teamColors = [
+        hcColor9.tweenTo(hcColor0, 0.25),
+        hcColor9.tweenTo(hcColor8, 0.65),
+        hcColor9.tweenTo(asHcColor(colors[3]), 0.85)
+    ],
+    teamSeries = Array(3).fill({
+        type: 'bubble',
+        shadow: true,
+        maxSize: '4%',
+        minSize: '1%',
+        point: {
+            events: {
+                mouseOver: function () {
+                    const chart = this.series.chart;
+                    chart.subtitle.element.style.opacity = 0;
+                    chart.setMidPaneBg({
+                        backgroundColor: teamColors[this.series.index],
+                        innerRadius: '0%'
+                    });
+                },
+                mouseOut
+            }
+        },
+        colorKey: 't',
+        tooltip: {
+            headerFormat: (
+                '<div class="team-day-display center">' +
+                '<span style="margin-bottom: 6em;">' +
+                '<b style="font-size: 1.4em; color:#000;">Day {point.x}</b>' +
+                '</span><span style="width:100%;' +
+                'margin-top:-130px;background: transparent; ' +
+                'font-size: 2em; padding: 0.8rem;' +
+                'border: 0 outset {series.color}; border-block-end:' +
+                '0 outset {series.color};"><b>{series.name}</b></span>'
+            ),
+            pointFormat: (
+                '<span style="margin-top:11em; position: absolute;' +
+                'font-size: 1em;"><span style="width:100%;' +
+                'text-align:center;">Daily Sales:</span></br>' +
+                '<span style="line-height:3em;width:100%;font-size:2em;' +
+                'text-align:center;">{point.z}</span>'
```

Comment:
Replace the inline styles with CSS class names perhaps? It would make these format strings look friendlier.

Reply:
True! I was a bit unsure about creating classes for the css-configurations which only occurred once, but I agree, it will look a lot nicer.

Reply:
Also, @TorsteinHonsi , regarding this:
I do not see a simple way to avoid the style attribute when using HCs format-string variables (`series.color`, `point.x.color` etc), so I assume its okay to keep styles using colors (there are a couple of cases involving `border-*`, there is the general `color` and `background-color`)

Reply:
Yes, that's fine!